app/views/home/pricing.html.erb

```diff
@@ -1,2 +1,37 @@
-<h1>Home#pricing</h1>
-<p>Find me in app/views/home/pricing.html.erb</p>
+<section class="pricing-header bg-light px-3 py-3 pt-md-5 pb-md-4 mx-auto text-center">
+  <h1 class="display-4">Plans</h1>
+  <h2 class="text-muted">Get started now on opeNode!</h2>
+  <p class="lead">We provide a free sandbox to get started and you will not have to enter your credit card information.<br/>We accept both Paypal and Cryptocurrency (Bitcoin, Bitcoin Cash, etc.) payments.</p>
```

Comment:
could be better that this part

> We provide a free sandbox to get started and you will not have to enter your credit card information.
We accept both Paypal and Cryptocurrency (Bitcoin, Bitcoin Cash, etc.) payments.

is left aligned (both We being aligned together), and also if the text is longer it should probably not take all space but change line automatically